File: output/qwen3.5-9b_IO_1/avatar/Python/Java/atcoder_ABC042_A.java
import java.util.Scanner;

public class atcoder_ABC042_A {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int[] ABC = new int[3];
        for (int i = 0; i < 3; i++) {
            ABC[i] = sc.nextInt();
        }
        
        int count5 = 0;
        int sum = 0;
        for (int num : ABC) {
            if (num == 5) {
                count5++;
            }
            sum += num;
        }
        
        if (count5 == 2) {
            if (sum == 17) {
                System.out.println("YES");
            } else {
                System.out.println("NO");
            }
        } else {
            System.out.println("NO");
        }
    }
}

File: output/qwen3.5-9b_IO_1/avatar/Java/Python/atcoder_ABC042_A.py
x, a, y = map(int, input().split())
h = "NO"
if (x == 5 or x == 7 or a == 5 or a == 7 or y == 5 or y == 7) and (x + y + a == 17):
    h = "YES"
print(h)
